Gem/Code/Source/Components/NetworkMatchComponent.cpp

```diff
@@ -6,9 +6,35 @@
  */
 
 #include <Source/Components/NetworkMatchComponent.h>
+#include <MultiplayerSampleTypes.h>
+#include <UiGameOverBus.h>
 
 namespace MultiplayerSample
 {
+    void NetworkMatchComponent::Reflect(AZ::ReflectContext* context)
+    {
+        AZ::SerializeContext* serializeContext = azrtti_cast<AZ::SerializeContext*>(context);
+        if (serializeContext)
+        {
+            serializeContext->Class<NetworkMatchComponent, NetworkMatchComponentBase>()
+                ->Version(1);
+        }
+        NetworkMatchComponentBase::Reflect(context);
+    }
+
+    void NetworkMatchComponent::HandleRPC_EndMatch(
+        [[maybe_unused]] AzNetworking::IConnection* invokingConnection, [[maybe_unused]] const MatchResultsSummary& results)
+    {
+        AZ_TracePrintf("NetworkMatchComponent", "Handling RPC_EndMatch\n", "...");
```

Comment:
Did you want to leave in the prints? Could be useful, but since the UI is already working this looks leftover from debugging.